--- profile.py	2019-09-23 14:15:15.350249556 +0200
+++ profile_new.py	2019-09-23 14:14:39.181259539 +0200
@@ -71,7 +71,7 @@
 
 # Other statements
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, '102X_dataRun2_HLT_v2', '')
+process.GlobalTag = GlobalTag(process.GlobalTag, '102X_upgrade2018_design_v9', '')
 
 # Path and EndPath definitions
 process.raw2digi_step = cms.Path(process.RawToDigi_pixelOnly)
@@ -101,8 +101,6 @@
 
 # End of customisation functions
 #do not add changes to your config after this point (unless you know what you are doing)
-from FWCore.ParameterSet.Utilities import convertToUnscheduled
-process=convertToUnscheduled(process)
 
 
 # Customisation from command line
